packages/open-payments/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "open-payments",
+  "description": "",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist/**/*"
+  ],
+  "scripts": {
+    "build": "pnpm clean && tsc --build tsconfig.json",
+    "clean": "rm -fr dist/",
+    "prepack": "pnpm build",
+    "generate:types": "npx ts-node scripts/generate-types.ts"
+  },
+  "devDependencies": {
+    "@types/node": "^18.7.12",
+    "openapi-typescript": "^4.5.0",
```

Comment:
I wrested quite a bit with this, but I wasn't able to use the newest version of this library. Using the programmatic way of generating types via `openapiTS` anything above v5 does not work: `ERR_REQUIRE_ESM`. Versions 5 and above do not support backwards compatibility with CommonJS: given the fact that having `type: module` in `package.json` break using this package as a workspace in the `/backend` project